Library/.rubocop.yml

```diff
@@ -6,6 +6,12 @@ AllCops:
     - '**/Casks/**/*'
     - '**/vendor/**/*'
 
+require:
+  - ./Homebrew/rubocops/bottle_block_cop.rb
```

Comment:
> but then we'd have to place all the cops in a single huge `rubocops.rb`

Not really, we just require all of the individual cops inside `rubocops/` with this file, so we don't have to do that inside the YAML file.